src/test/gen/crypto_gen.cpp

```diff
@@ -14,6 +11,13 @@
 rc::Gen<std::vector<CKey>> MultisigKeys()
 {
     return rc::gen::suchThat(rc::gen::arbitrary<std::vector<CKey>>(), [](const std::vector<CKey>& keys) {
+        //TODO: Investigate why we can only allow 15 keys. Consensus rules
+        // dictate we can up to 20 keys
+        //https://github.com/bitcoin/bitcoin/blob/10bee0dd4f37eb6cb7a0f1d565fa0fecf8109c35/src/script/script.h#L29
```

Comment:
the wallet has no idea how to make a script using numbers beyond OP_16. Once you do n-of-17 you need a single byte pushing instruction to push the number `17`

Reply:
I think the original reason for this was actually that more than 15 public keys it was too big of a val to push onto the stack (520 byte limit)